moped-editor/src/utils/dataGridHelpers.js

```diff
@@ -2,8 +2,15 @@ import { GridRowEditStopReasons } from "@mui/x-data-grid-pro";
 
 export const defaultEditColumnIconStyle = { fontSize: "24px" };
 
-export const handleRowEditStop = (params, event) => {
+export const handleRowEditStop = (rows, setRows) => (params, event) => {
```

Comment:
Cool! A curried function works nicely here. I had a small concern that `rows` could get stale, but I did some testing and the innermost function is redefined every render so the state remains up to date.

Fun to think through this one! 🙏